app/rpc/namespaces/eth/api.go

```diff
@@ -1206,14 +1206,20 @@ func (api *PublicEthereumAPI) GetTransactionReceipt(hash common.Hash) (*watcher.
 		contractAddr = nil
 	}
 
+	// fix gasUsed when deliverTx ante handler check sequence invalid
```

Comment:
```go
res, e := api.wrappedBackend.GetTransactionReceipt(hash)
```
the Receipt of transaction which was returned by watcher need to resolve the wrong GasUsed